WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -132,6 +137,13 @@ sealed class PostListItemViewHolder(
         }
     }
 
+    private fun updatePostInfoLabel(view: TextView, uiStrings: List<UiString>?) {
+        val concatenatedText = uiStrings?.joinToString(separator = "  ·  ") {
```

Comment:
I decided to move the concatenation from the PostListItemUiStateHelper to the view holder. This isn't ideal, but since we don't have access to the context in the PostListItemUiStateHelper we can't transform post status UiStringRes to the actual string.